### DISCUSSION TITLE
Validating files from a network share

### DISCUSSION THREAD

**Discussion by fferguson10** [CMSgov/price-transparency-guide-validator#56] [General]
Has anyone had any issues validating files from a network share? When using the validate command with a share folder path (i.e. cms-mrf-validator validate \\{severname}\{folder}\{fille.json} v1.0.0 -t allowed-amounts). The command returns a "file not found" error. I've also tried placing the file share path in quotes, same result.

**Comment by olderthandirt5** [comment]
I have tried several different combinations with similar results, both from an administrator command prompt and a powershell prompt. If there are spaces in the path, quotes are required but even with quotes I have not been able to validate a network share.
I also tried a 'mapped' drive letter with same results.
I think it might be a Docker security issue. I am a little rusty on configuring a Docker container to reach a network asset, but I am pretty sure I have done it in the past.
Most helpful error message I have received is "docker: Error response from daemon: \\servername\folder\folder2\folder3%!(EXTRA string=is not a valid Windows path).
See 'docker run --help'."

If you have access to a Docker resource, I would start there based on the dir command working with the same path.

**Reply by fferguson10** [reply]
Thanks for the response. I'll give that a try.
